plugins/jobs/web_client/views/timeChartConfig.js

```diff
@@ -270,7 +270,7 @@ export default {
     'legends': [
         {
             'fill': 'timing',
```

Comment:
No icon. It looks like your screenshot, but without a set of icon keys, it's just a list of words. I'm not sure what the purpose is on this plot.